docs: Update Placeholder Image and Lorem Ipsum shortcode pages (#1105)
quarto-dev/quarto-web: docs/authoring/lipsum.qmd

## Overview

> When you need to visualize how text will look on your website, the `{{{< lipsum >}}}` shortcode comes in handy.

Reviewer: ```suggestion
When you need to visualize how text will look in your document or your website, the `{{{< lipsum >}}}` shortcode comes in handy.
```

## Usage

::: {layout-ncol=1}
```markdown
{{{< lipsum 1 >}}}
```
:::: {.border}
{{< lipsum 1 >}}
::::
:::

> `lipsum` takes a single parameter, in two possible formats:

Reviewer: Pretty sure this parameter is optional:
```suggestion
`lipsum` takes an optional parameter, in two possible formats:
```
